frontend/common/src/components/tabs/Tab.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+
+export type TabColor = "primary" | "secondary";
+type TabAppearance = "active" | "inactive";
+
+const styleMap: Record<
+  TabColor,
+  Record<TabAppearance, Record<string, string>>
+> = {
+  primary: {
+    active: {
+      "data-h2-font-color": "b(lightpurple)",
+      "data-h2-font-weight": "b(bold)",
+    },
+    inactive: {
+      "data-h2-font-color": "b(black)",
+    },
+  },
+  secondary: {
+    active: {
+      "data-h2-font-color": "b(lightnavy)",
+      "data-h2-font-weight": "b(bold)",
+    },
+    inactive: {
+      "data-h2-font-color": "b(black)",
+    },
+  },
+};
+export interface TabProps extends React.HTMLProps<HTMLElement> {
+  icon?: JSX.Element;
+  iconOpen?: JSX.Element;
+  iconClosed?: JSX.Element;
+  iconPosition?: "left" | "right";
+  text?: string;
+  variant?: "normal" | "close" | "label";
```

Comment:
I'm not 100% happy with this prop name or its options.  Open to suggestions.

Reply:
Taking a stab at this, I tried to go through and see if I could offer advice.

How about
tabType?: "header" | "closer" | "label"

Reply:
Whatever it's named, a comment might help explain its purpose.

Reply:
Thanks, guys.  I took some of Vachan's suggestions and of course it should be documented. 😳

Reply:
yeah it does sound like  covid-19. tabType sounds better.